package-lock.json

```diff
@@ -682,6 +682,12 @@
         "@sinonjs/commons": "^1.7.0"
       }
     },
+    "@tootallnate/once": {
```

Comment:
I see that the `package.json` didn't change in terms of dependencies, why did the `package-lock.json` changed so much?